### PR TITLE
feat: Add Last updated on to each item in the list

### PR DESCRIPTION
**Pull request by harishkotra**
- [x] I've checked that this isn't a duplicate pull request.

Added 'Last updated on' for each item in the list based on the dateAdded from data.json as per the issue #266

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9909,16 +9909,13 @@
           "resolved": "http://registry.npmjs.org/got/-/got-6.7.1.tgz",
```
Comment (aslafy-z) [inline package-lock.json]: Could you please reset this file?

Comment (harishkotra) [reply package-lock.json]: @aslafy-z do you mean to revert it to one in the master repo?

Comment (vikaspotluri123) [reply package-lock.json]: Yep, since you haven't made any updates to the deps, this file shouldn't change 😄

**Comment by vikaspotluri123** [comment]
Suggested UI

Thoughts on this?

**Comment by harishkotra** [comment]
@vikaspotluri123 I like it. I'll realign it to one single line and since Check it out won't change, it shouldn't affect the layout and will still look similar to what we have right now. 

Should we close this PR until I submit a fresh PR with the new layout you suggested?

**Comment by vikaspotluri123** [comment]
You can just add commits to this PR 😄

**Comment by harishkotra** [comment]
@vikaspotluri123 getting there with this git game. Thanks. 

Pushed the changes.

**Comment by harishkotra** [comment]
@vikaspotluri123 Can we merge this?

**Comment by swapagarwal** [comment]
@harishkotra Can you remove changes from data.json?

**Comment by harishkotra** [comment]
> @harishkotra Can you remove changes from data.json?

Done.

File: src/styl/imports/card.styl [hunk 2]
```diff
@@ -62,3 +62,5 @@
 		width 100%
 		overflow hidden
 		padding-bottom 15px
+	.last-updated
```
Comment (aslafy-z) [inline src/styl/imports/card.styl]: Please add a line break just before :+1:

**Comment by vikaspotluri123** [comment]
Please rebase so we can run UI sanity checks as well 💃

**Comment by harishkotra** [comment]
> Please rebase so we can run UI sanity checks as well 💃

@vikaspotluri123 done :)

**Comment by vikaspotluri123** [comment]
Doesn't seem to have worked... did you force push?

**Comment by harishkotra** [comment]
> 
> 
> Doesn't seem to have worked... did you force push?

Nope. There is an image URL in data.json that stopped working. I just did a rebase, updated the style as per @aslafy-z 's comment and pushed it. Didn't do any force push.

**Comment by vikaspotluri123** [comment]
> I just did a rebase

If you did a rebase, you won't be able to update your branch unless you force push

**Comment by aslafy-z** [comment]
`date added` != `last updated`. @vikaspotluri123 we need to work on this

**Comment by vikaspotluri123** [comment]
For now we can change it to `added`... When we add the `updated` "column" to the "database", we can update this as well. I want to wait for hacktoberfest to be over before making changes like this so we can limit the number of PRs that have to be rebased

**Pull request closed without merging**